Fixes

include/gtk-2.0/gtk/gtkitemfactory.h:47:1: warning: function declaration isn't a prototype [-Wstrict-prototypes]
 typedef void (*GtkItemFactoryCallback)  ();

gcc5 has -Wstrict-prototypes on by default for -Werror so this becomes a build failure for consumers
of this header e.g. matchbox-panel-2

Upstream-Status: Pending

Signed-off-by: Khem Raj <raj.khem@gmail.com>
Index: gtk+-2.24.27/gtk/gtkitemfactory.h
===================================================================
--- gtk+-2.24.27.orig/gtk/gtkitemfactory.h
+++ gtk+-2.24.27/gtk/gtkitemfactory.h
@@ -44,7 +44,7 @@ typedef void	(*GtkPrintFunc)		   (gpoint
  * (Note that if we are included from a C++ program () will mean
  * (void) so an explicit cast will be needed.)
  */
-typedef	void	(*GtkItemFactoryCallback)  ();
+typedef	void	(*GtkItemFactoryCallback)  (void);
 typedef	void	(*GtkItemFactoryCallback1) (gpointer		 callback_data,
 					    guint		 callback_action,
 					    GtkWidget		*widget);
